readme.md

```diff
@@ -0,0 +1,157 @@
+--to, -t       Email address you are sending to            [string] [required]
+--subject      The subject line of the email               [string] [required]
+--html         The HTML version of the email                          [string]
+--text         The text version of the email                          [string]
+```
+
+### template [options]
+
+Send a templated email.
+
+```bash
+$ postmark email template --alias="" --from="" --to="" --model='{"name": "Jane"}'
+
+# Options
+--id, -i     Template ID                                              [string]
+--alias, -a  Template Alias                                           [string]
+--from, -f   Email address you are sending from. Must be an address on a
+             verified domain or confirmed Sender Signature.[string] [required]
+--to, -t     Email address you are sending to              [string] [required]
+--model, -m                                                           [string]
+```
+
+## postmark servers _&lt;command&gt;_
+
+### list [options]
+
+Fetch a list of servers on an account.
+
+```bash
+$ postmark servers list
+
+# Options
+--count, -c   Number of servers to return                             [number]
+--offset, -o  Number of servers to skip                               [number]
+--name, -n    Filter servers by name                                  [string]
+```
+
+## postmark templates _&lt;command&gt;_
+
+Provides a push and pull workflow for your email templates. This lets you store your templates in version control and push to Postmark via a CI/CD pipeline.
+
+**⚠️ Before you get started**, make sure that all of your templates have an alias. Check out our [help doc](https://postmarkapp.com/support/article/1117-how-do-i-use-a-template-alias) for more info.
+
+### pull _&lt;output directory&gt;_ [options]
+
+Download templates from a Postmark server to your local file system.
+
+```bash
+$ postmark templates pull ~/Desktop/my_templates
+
+#Options
+--overwrite, -o  Overwrite templates if they already exist
+                                                    [boolean] [default: false]
+```
+
+Here’s an example of the directory structure that is downloaded:
+
+```bash
+my_templates
+├── password_reset
+│   ├── content.html
+│   ├── content.txt
+│   └── meta.json
+└── welcome
+    ├── content.html
+    └── meta.json
+```
+
+Each template is stored in its own subdirectory with the HTML and text content, and metadata. The metadata contains the template’s name, subject, and alias.
+
+**NOTE:** If you update the template alias in `meta.json`, be sure to rename the template’s directory so it matches. This prevents conflicts down the line when pulling templates.
+
+### push _&lt;templates directory&gt;_ [options]
+
+Pushes templates from the local file system to a Postmark server.
+
+```bash
+$ postmark templates push ~/Desktop/my_templates
+
+# Options
+--confirmation, -c  Require user confirmation before pushing the templates
```

Comment:
Ditto about using a boolean flag here. It would renaming to something like `--no-confirmation` though.

Reply:
Great point. I went down the path of updating this to `--no-confirmation`, but yargs is having issues interpreting the options that contain a dash in the name. I'm guessing it's a bug 😕

I ended up renaming this option to `--force` since this basically force pushes the templates. I feel like it plays nicely since we're sticking to a git analogy here.